pkg/controller/cluster/kes-csr.go

```diff
@@ -48,7 +48,7 @@ func generateKESCryptoData(mi *miniov1.MinIOInstance) ([]byte, []byte, error) {
 	csrTemplate := x509.CertificateRequest{
 		Subject: pkix.Name{
 			CommonName:   mi.KESWildCardName(),
-			Organization: miniov1.DefaultOrgName,
+			Organization: mi.Spec.CertConfig.OrganizationName,
```

Comment:
It may be confusing to use the org name passed for MinIO pods for KES cert. We can bring this up in the CRD spec so it is clear.

Reply:
We use a single AutoCert Flag to generate all the certs and they are all for the same customer provided metadata. This is technically the same Org's certificate.